expose as zig pkg artifact

lun-4 opened the pull request.

lun-4 committed:
expose as zig pkg artifact

lun-4 committed:
install pcre.h

nektro commented:
merging this is blocked on https://github.com/ziglang/zig/issues/17818 because i dont want to break the zigmod workflow yet

lun-4 committed:
remove temporary hack

kivikakk committed:
build.zig: use b.path(..) instead of LazyPath anon structs.

Builds on 0.12 and 0.13.

kivikakk committed:
build.zig: don't copy these around.

It's confusing and they're unused (and shouldn't be used).

kivikakk committed:
config.h: set NO_RECURSE.

It's not too hard to cause a crash on darwin-arm64 without this; see
https://github.com/kivikakk/libpcre.zig/pull/23.

lun-4 committed:
Merge pull request #1 from kivikakk/zigpkg-0.13

build.zig: use b.path(..) instead of LazyPath anon structs.
